### PR TITLE
Add deprecations and removals for Chrome 92.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by jpmedley** [comment]
@robdodson I didn't realize this was still open. What I need is a copy edit from @rachelandrew or @mihajlija.

**Comment by myakura** [comment]
> What I need is a copy edit

Is that the same reason that deps-rems-91 post is holding off? I see there is a branch but there hasn't been a PR yet.
https://github.com/GoogleChrome/developer.chrome.com/issues/833#issuecomment-850822367

File: site/en/blog/deps-rems-92/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Deprecations and removals in Chrome 92
+description: >
+  A round up of the deprecations and removals in Chrome 92 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2021-06-03
```
Comment (rachelandrew) [inline site/en/blog/deps-rems-92/index.md:6]:
```suggestion
date: 2021-07-23
```

File: site/en/blog/deps-rems-92/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Chrome 92 beta was released on June 3, 2021 and is expected to become the
+stable version in late July, 2021.
+
+## Payment handlers for standardized payment method identifiers
+
+This feature, which enabled web-based payment handlers to receive paymentrequest
+events with non-URL, but standardized payment method identifiers, such as
+`"basic-card"` or `"tokenized-card"`, [has been
+removed](https://www.chromestatus.com/feature/5407573287108608).
```
Comment (rachelandrew) [inline site/en/blog/deps-rems-92/index.md:23]:
```suggestion
removed](https://www.chromestatus.com/feature/5717324021628928).
```

Comment (rachelandrew) [reply site/en/blog/deps-rems-92/index.md:23]: I think this is the correct URL.

File: site/en/blog/deps-rems-92/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+{% include 'partials/see-all-dep-rem.md' %}
+
+Chrome 92 beta was released on June 3, 2021 and is expected to become the
+stable version in late July, 2021.
+
+## Payment handlers for standardized payment method identifiers
+
+This feature, which enabled web-based payment handlers to receive paymentrequest
```
Comment (rachelandrew) [inline site/en/blog/deps-rems-92/index.md:20]:
```suggestion
This feature, which enabled web-based payment handlers to receive `paymentrequest`
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]

**Comment by jpmedley** [comment]
@rachelandrew PTAL

**Comment by samthor** [comment]
Can we get this in? I'm happy with it, although not a writer.

**Review by rachelandrew — APPROVED** [review]

File: site/en/blog/deps-rems-92/index.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+---
+title: Deprecations and removals in Chrome 92
+description: >
+  A round up of the deprecations and removals in Chrome 92 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2021-07-23
```
Comment (rachelandrew) [inline site/en/blog/deps-rems-92/index.md:6]:
```suggestion
date: 2021-08-16
```

**Pull request merged**